libslide/src/bignum/mul.rs

```diff
@@ -28,7 +28,7 @@ impl ops::Mul for Bignum {
             rhs_int.push(0);
             lhs_int.push(0);
         }
-        while !ispowerof2(rhs_int.len() + rhs_dec.len()) {
+        while !ispowerof2(rhs_int.len() + rhs_dec.len()) || rhs_int.len() <= 2 {
             rhs_int.push(0);
             lhs_int.push(0);
```

Comment:
why do we push to both rhs and lhs

Reply:
Preserving even length from what I can see?